api/src/main/java/org/apache/gravitino/credential/Credential.java

```diff
@@ -29,6 +29,9 @@ public interface Credential {
   /** Credential expire time in ms since the epoch. */
   String EXPIRE_TIME_IN_MS = "expire-time-in-ms";
 
+  /** The HTTP header used to get the credential from fileset location */
+  String HTTP_HEADER_CURRENT_LOCATION_NAME = "CurrentLocationName";
```

Comment:
We'd better not use camel style string for http header property.

Reply:
I just follow the existing code style:
https://github.com/apache/gravitino/blob/bd5fad2adb0c4c2383fc03b812d132459ae6e4ca/common/src/main/java/org/apache/gravitino/audit/FilesetAuditConstants.java#L26-L30

should I change them together in this PR or just change current header to `Current-Location-Name`?

Reply:
No, it will break the compatibility, let me think more on this.

Reply:
Please look at this https://stackoverflow.com/questions/5258977/are-http-headers-case-sensitive.

Reply:
Although HTTP header names are not case sensitive technically, it is customary to use Pascal Case With Hyphens format, such as:
```
Content-Type
Accept-Language
Cache-Control
```

Reply:
Right. It is case insensitive, so the convention was to use hyphens.